open-metadata-conformance-suite/open-metadata-conformance-suite-api/src/main/java/org/odpi/openmetadata/conformance/ffdc/ConformanceSuiteErrorCode.java

```diff
@@ -234,6 +234,18 @@ public String getUnformattedErrorMessage()
         return errorMessage;
     }
 
+    /**
+     * Returns the string but with any special storage characters changes to underscores
+     *
+     * @return String
+     */
+    private String sanatiseLogString( String str ) {
```

Comment:
Suggest 'sanitize' or 'sanitise' for spelling, though only cosmetic

Reply:
@planetf1 The scan results will really help and show most cases as obvious, this is the report I'm using and it's still available, I've just checked.

`https://sonarcloud.io/project/issues?resolved=false&sonarsourceSecurity=log-injection&id=odpi_egeria&open=AXnX8Vhw9ycZkvF8HLgx` 

most of these cases compared references to collection that had been "new" above.
Some where nested inside the same null check.
One was a duplicate check in the same 'if' 

if it helps with the confidence I've noticed that I get 'possible null pointer' alerts from one of the other scanners and this PR is coming in clean. 

While code will change continually I would expect Egeria use of 'null' to reduce rather than increase, the Java community seem to advocate empty lists and default values.

Reply:
Yes, I'm ok with your proposed changes. It was a good challenge from the tools.